src/admin-views/aggregator/origins/url.php

```diff
@@ -69,6 +69,13 @@ class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tr
 		></span>
 	</td>
 </tr>
+
+<?php
+if ( 'edit' === $tab->get_slug() ) {
+	$this->template( 'fields/schedule', array( 'record' => $record, 'origin' => $origin_slug, 'aggregator_action' => $aggregator_action ) );
```

Comment:
This is a long line. Maybe:

```php
$this->template(
  'fields/schedule',
  array(
    'record'            => $record,
    'origin'            => $origin_slug,
    'aggregator_action' => $aggregator_action,
  )
);
```

or

```php
$this->template( 'fields/schedule', array(
  'record'            => $record,
  'origin'            => $origin_slug,
  'aggregator_action' => $aggregator_action,
) );
```

But...it doesn't _need_ to change. Just a thought.